packages/forms-web-app/__tests__/unit/controllers/full-planning/out-of-time-shutter-page.spec.js

```diff
@@ -0,0 +1,33 @@
+const outOfTimeShutterPageController = require('../../../../src/controllers/full-planning/out-of-time-shutter-page');
+const { mockReq, mockRes } = require('../../mocks');
+const { VIEW } = require('../../../../src/lib/views');
+
+jest.mock('../../../../src/lib/logger');
+
+describe('controllers/full-planning/out-of-time-shutter-page', () => {
+  let req;
+  let res;
+
+  beforeEach(() => {
+    req = mockReq();
+    res = mockRes();
+
+    jest.resetAllMocks();
+  });
+
+  describe('getOutOfTimeShutterPage', () => {
```

Comment:
Remove this describe and move under the first describe.

Reply:
done